api/git/azure.go

```diff
@@ -54,6 +59,18 @@ type azureClient struct {
 }
 
 func NewAzureClient() *azureClient {
+	azureDevOpsServerHost, azureDevOpsServerHostOK := os.LookupEnv("AZURE_DEVOPS_SERVER_HOST")
+	azureDevOpsServerBaseUrl, azureDevOpsServerBaseUrlOK := os.LookupEnv("AZURE_DEVOPS_SERVER_BASE_URL")
+	if azureDevOpsServerHostOK && azureDevOpsServerBaseUrlOK {
+		return &azureClient{
+			baseUrl: "https://" + azureDevOpsServerHost + "/" + azureDevOpsServerBaseUrl,
```

Comment:
Do we need to force to https?
Also I would use the whole url as the env entry as the name of the variable is ..baseurl (for example http://host/:api). Otherwise its a bit confusing to know what to put there.

Reply:
I've changed the logic to only get a single AZURE_DEVOPS_SERVER_URL env with {protocol}://{host}/{customized_address} format. This should clear any confusion that users might have about address formating.